### PR TITLE
CTOR-924 : Refresh network-routers-atrica-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-routers-atrica-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 1]
```diff
@@ -2,48 +2,376 @@
...
+* **Net-Atrica-SNMP-custom**
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">
+
+| Alias | Modèle de service | Description |
+|:------|:------------------|:------------|
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:23]: nothing

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 2]
```diff
@@ -2,48 +2,376 @@
...
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Atrica-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                    | Modèle de service                               | Description          |
+|:-------------------------|:------------------------------------------------|:---------------------|
+| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le traffic  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:32]:
```suggestion
| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 3]
```diff
@@ -2,48 +2,376 @@
...
+| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le traffic  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:33]:
```suggestion
| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 4]
```diff
@@ -2,48 +2,376 @@
...
+| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le traffic  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:34]:
```suggestion
| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |
```

File: pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 5]
```diff
@@ -2,48 +2,375 @@
...
+
+* **Net-Atrica-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">
+
+| Service Alias | Service Template  | Service Description |
+|:--------------|:------------------|:--------------------|
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:22]: nothing

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md (hunk 3 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:33]: we have 3 times the same description for 3 different things

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md (hunk 1 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:23]:
```suggestion
Pas de modèle de service rattaché à ce modèle d'hôte.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 6]
```diff
@@ -2,48 +2,376 @@
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Atrica-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                    | Modèle de service                               | Description          |
+|:-------------------------|:------------------------------------------------|:---------------------|
+| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic  |
+| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic  |
+| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:34]:
```suggestion
| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic de l'équipement ayant un ID spécifié |
| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic de l'équipement ayant un nom spécifié |
| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |
```

File: pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md (hunk 5 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:22]:
```suggestion
No service template linked to this host template.
```

File: pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 7]
```diff
@@ -2,48 +2,375 @@
...
+> The services listed above are created automatically when the **Net-Atrica-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias            | Service Template                                | Service Description |
+|:-------------------------|:------------------------------------------------|:--------------------|
+| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Check traffic       |
+| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Check traffic       |
+| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Check traffic       |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:33]:
```suggestion
| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Check device traffic using a specified ID       |
| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Check device traffic using a specified name      |
| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Check traffic       |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 8]
```diff
@@ -2,48 +2,375 @@
...
+<TabItem value="Connections-Global" label="Connections-Global">
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+| Macro                 | Description                                                                                                                                                                                                                                                                            | Valeur par défaut  | Obligatoire |
+|:----------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|
+| FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                                                                                                                                                                              | .+                 |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:236]:
```suggestion
| FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all connections'). Can be a regex                                                                                                                                                                                              | .+                 |             |
```

File: pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 9]
```diff
@@ -2,48 +2,374 @@
...
 
-### SNMP Permissions
+</TabItem>
+<TabItem value="Connections-Global" label="Connections-Global">
 
-Read-Only access.
+| Macro                 | Description                                                                                                                                                                                                                                                                            | Default value      | Mandatory   |
+|:----------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|
+| FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                                                                                                                                                                              | .+                 |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:237]:
```suggestion
| FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all connections'). Can be a regex                                                                                                                                                                                             | .+                 |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 10]
```diff
@@ -2,48 +2,375 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">
+
+Pas de modèle de service rattaché à ce modèle d'hôte.
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Atrica-SNMP-custom** est utilisé.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:24]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md [hunk 11]
```diff
@@ -2,48 +2,374 @@
...
+* **Net-Atrica-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">
+
+No service template linked to this host template.
+
+> The services listed above are created automatically when the **Net-Atrica-SNMP-custom** host template is used.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md:23]:
```suggestion
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**